app/controllers/sessions_controller.rb

```diff
@@ -9,7 +9,6 @@ def create
     @user = User.find_by(username: params[:username])
     if @user && @user.authenticate(params[:password])
 
-
         session[:user_id] = @user.id
 
         redirect_to '/welcome'
```

Comment:
It might make more sense to change welcome.html.erb to index.html.erb

Reply:
good point